terraform-post-packer/main.tf

```diff
@@ -24,8 +24,8 @@ locals {
   account_name_regex  = format("^env[[:digit:]]+ \\(%s\\)$", local.images_account_type)
 }
 
-# The most-recent AMI created by cisagov/skeleton-packer
-data "aws_ami" "example" {
+# The IDs of all cisagov/skeleton-packer AMIs
```

Comment:
The ID is an attribute of an AMI resource.

```suggestion
# The AMIs created by cisagov/skeleton-packer
```

Reply:
[`aws_ami`](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/ami) is a different data resource than [`aws_ami_ids`](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/ami_ids).  The latter returns only the IDs, not the complete AMI object.

Reply:
Ack I missed the resource type change, my apologies.